public/js/test/node-unit-tests.js

```diff
@@ -3,6 +3,8 @@
 const glob = require("glob").sync;
 const path = require("path");
 
+require("amd-loader");
```

Comment:
Neat. At this point, I wouldn't be surprised if we ended up using webpack for devtools generally so when that happens the JSON viewer can use that and we can avoid AMD completely.